docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -5,4 +5,677 @@ navigation:
+        {special && (
+          {/* Customized content for store-a */}
+          <>// [!code ++]
+             <span className="block text-red-700">Super offer!<span>// [!code ++] 
+             <span
+               className={classNames('font-normal text-neutral-500 line-through', classNameVariants?.special)}
+               data-testid="regular-price"
+             >
+               {special}
+             </span>
+          </>// [!code ++] 
+        )}
+      </div>
+    )
+  );
+}
+```
+
+#### In Nuxt
+
+The process is similar in Nuxt. Here’s an example:
+
+```vue
+// /apps/stores/store-a/storefront-unified-nuxt/components/DecoratedPrice/DecoratedPrice.vue
+
+<template>
+  <div class="flex gap-x-2 items-baseline">
+    <span
+      data-testid="special-price"
+      :class="[
+        'font-semibold',
+        classNameVariants?.regular,
+        {
+          <!-- Customized class for store-a -->
+          'text-neutral-900': !special, // [!code --]
+          'text-primary-700': !special, // [!code ++] 
+          'text-secondary-700': special,
+        },
+      ]"
+    >
+      {{ regular }}
+    </span>
+     <!-- Customized content for store-a -->
+    <span v-if="special" className="block text-red-700">Super offer!<span>// [!code ++] 
+    <span
+      v-if="special"
+      data-testid="regular-price"
+      :class="['font-normal text-neutral-500 line-through', classNameVariants?.special]"
+    >
+      {{ special }}
+    </span>
+  </div>
+</template>
+
+<script setup lang="ts">
+// Props remain unchanged
+defineProps<{
+   className?: string;
+   classNameVariants?: ClassNameVariants;
+   regular: string;
+   special?: string;
+}>();
+</script>
+```
+
+### Outcome
+
+In both cases (Next.js and Nuxt), props remain untouched, ensuring compatibility with the rest of the application. Adjustments remain restricted to layout or presentation, avoiding unnecessary duplication or breaking changes to functionality.
+
+## Breaking Changes in Component Markup
+
+Some changes may require adjustments to the component’s **props interface**, meaning that the component is no longer compatible with its previous usage. These are referred to as **breaking changes** because they can potentially impact other parts of the application if not handled correctly.
+
+### When to Use
+
+- When introducing additional props for brand-specific requirements.
+- When removing unused or deprecated props for cleaner, leaner implementations.
+- When creating a fundamentally different component design unique to specific brands or stores.
+
+### Example Workflow for Breaking Changes
+
+#### Scenario 1: Adding Props
+
+The `DecoratedPrice` requires a new prop `customText`, exclusively for `store-b`.
+
+1. **Duplicate** the component for `store-b`.
+2. **Modify the props interface** to include the new prop.
+3. **Implement fallback behaviors** it's best to make new props as optional to ensure existing components using the old interface remain functional
```

Comment:
I would make this statement stronger - it's not "best" to make props optional it's the only way :D 
```suggestion
3. **Implement fallback behaviors** make new props as optional to ensure existing components using the old interface remain functional
```

Reply:
Changed